ak_vendor/report_sample1.json

```diff
@@ -113,6 +113,38 @@
             "description": "Use strong cryptography and security protocols such as SSL/TLS, SSH or IPSec to safeguard sensitive cardholder data during transmission over open, public networks (e.g. Internet, wireless technologies, Global System for Mobile communications [GSM], General Packet Radio Service [GPRS]). Ensure wireless networks transmitting cardholder data or connected to the cardholder data environment use industry best practices (e.g., IEEE 802.11i) to implement strong encryption for authentication and transmission. The use of WEP as a security control is prohibited",
             "title": "Encrypt transmission of cardholder data across open, public networks"
           }
+        ],
+        "mstg": [
+          {
+            "code": "MSTG-NETWORK-1",
+            "title": "Data is encrypted on the network using TLS. The secure channel is used consistently throughout the app."
+          }
+        ],
+        "asvs": [
+          {
+            "code": "V1.1.1",
+            "title": "Verify the use of a secure software development lifecycle that addresses security in all stages of development."
+          },
+          {
+            "code": "V1.1.2",
+            "title": "Verify the use of threat modeling for every design change or sprint planning to identify threats, plan for countermeasures, facilitate appropriate risk responses, and guide security esting."
```

Comment:
spelling mistakes

Reply:
I've verified all the places where I copied the content and there is higher chance of spelling mistakes. But the contents in asvs.py is so huge that I can't even check it 😂  but the good thing is it was scraped using a script.